libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/CohortStatsHeatmap.tsx

```diff
@@ -97,6 +97,9 @@ export class CohortStatsHeatmap extends React.Component<
             min: 0,
             minColor
           },
+          custom: {

```

Comment:
two minor things:
1.) I think most of these properties belong in some styles file, and can be re-imported here
2.) "magic numbers" - is the 190 coming from somewhere else?  can we re-use the same variable to reference that value?